app/src/org/commcare/fragments/FileServerFragment.java

```diff
@@ -101,8 +99,7 @@ public void startServer(String mReceiveZipDirectory) {
         private final FileServerFragment mListener;
         private boolean socketOccupied;
 
-        public FileServerAsyncTask(FileServerFragment mListener) {
-            Log.d(TAG, "new fileasync task");
+        FileServerAsyncTask(FileServerFragment mListener) {
```

Comment:
why remove the `public` modifier?

Reply:
Lint was complaining because this is only used inside this class